yandex_contest/binary_sum.py

```diff
@@ -1,44 +1,35 @@
-def convert(string):
-    list_1 = []
-    list_1[:0] = string
-    list_x = [int(i) for i in list_1]
+def convert_to_list(string):
+    list_x = [int(i) for i in string]
     return list_x
 
+
 def main():
     num_1 = input()
-    list_1 = convert(num_1)
+    list_1 = convert_to_list(num_1)
 
     num_2 = input()
-    list_2 = convert(num_2)
+    list_2 = convert_to_list(num_2)
+
+    reversed_num_1 = list(reversed(list_1))
+    reversed_num_2 = list(reversed(list_2))
+
+    if len(reversed_num_1) > len(reversed_num_2):
+        temp = reversed_num_1
+        reversed_num_1 = reversed_num_2
+        reversed_num_2 = temp
```

Comment:
There is a trick in python to do such swap operation in a much nicer way:
```suggestion
    if len(reversed_num_1) > len(reversed_num_2):
        reversed_num_1, reversed_num_2 = reversed_num_2, reversed_num_1
```